webapp/src/shared/components/Input/EditUserForm.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from 'react';
+import { Button, ButtonVariant, Input, InputVariant } from '../';
+import './EditUserForm.css';
+
+export default function EditUserForm() {
+  const [fullName, setFullName] = useState('');
+  const [email, setEmail] = useState('');
+
+  const buttonAction = (e: React.FormEvent<HTMLFormElement>): void => {
+    e.preventDefault();
+    alert(`Your name: "${fullName}" and your email: "${email}" were saved.`);
+  };
+  return (
+    <form className="edit-user-form" onSubmit={buttonAction}>
+      <Input
+        variant={InputVariant.LIGHT}
+        label="Full Name"
+        placeholder="Full Name"
+        value={fullName}
+        name="fullName"
+        onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+          setFullName(e.target.value);
+        }}
```

Comment:
```suggestion
        onChange={(e) => setFullName(e.target.value)}
```

suggestion: We could omit the parameter type because when the handler is inline, TypeScript can infer it.
Also, we could remove the parenthesis from the method body because it's one line.

Same suggestion for the email input.

Reply:
Done